### PR TITLE
Fix exiting edit mode by clicking anywhere

### LINKED ISSUE #678: Exiting the Edit Background Mode
## Bug Description

When I enter the “edit background” mode, I struggle to exit. I naturally try to exit by clicking outside into the black area but that doesn't exit. I have to click on the greyed out areas of the image to exit. I feel like clicking on anything other than the image should exit the background edit mode. 

image

## Screenshots

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by wassgha**
Closes #678

### CODE REVIEW

File: assets/src/edit-story/components/canvas/navLayer.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ import {
 
 function NavLayer() {
   return (
-    <Layer pointerEvents="none" onMouseDown={(evt) => evt.stopPropagation()}>
```
Comment (wassgha) [inline assets/src/edit-story/components/canvas/navLayer.js]: @dvoytenko I think you added this, any idea what could go wrong here if we remove it?

Comment (dvoytenko) [reply assets/src/edit-story/components/canvas/navLayer.js]: The things I'd check:
1. Starting lasso selection where we don't want (page carousel, page toolbar, etc)
2. Clearing out selection where we don't want

**Comment by dvoytenko** [comment]
Maybe a safer option would be to exit edit mode when someone is clicking on the gray area of the edit layer? Also, we have requirements to allow the user to click on some elements in the design panel w/o exiting edit mode (e.g. "Bold" button while editing text).

**Comment by wassgha** [comment]
Ready

**Review by dvoytenko — APPROVED** [review]

**Pull request merged**